tests/unittest/_torch/modules/mamba/test_flashinfer_gdn_decode.py

```
@@ -0,0 +1,109 @@
+
+
+def _fi_decode_available() -> bool:
+    if not torch.cuda.is_available():
+        return False
+    from tensorrt_llm._utils import is_flashinfer_gdn_supported_arch
+
+    if not is_flashinfer_gdn_supported_arch():
+        return False
+    try:
+        from flashinfer.gdn_kernels.gdn_decode_bf16_state import gated_delta_rule  # noqa: F401
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)